tests/providers/aws/aws_provider_test.py

```diff
@@ -1370,7 +1369,7 @@ def test_test_connection_with_role_from_env_invalid_session_name(self):
         assert exception.type == AWSArgumentTypeValidationError
         assert (
             exception.value.args[0]
-            == "[1905] AWS argument type validation error - Role Session Name must be between 2 and 64 characters and may contain alphanumeric characters, periods, hyphens, and underscores."
+            == "[1905] Session Duration must be between 900 and 43200 seconds. - Role Session Name must be between 2 and 64 characters and may contain alphanumeric characters, periods, hyphens, and underscores."
```

Comment:
This is strange... this exception should not refer to the session duration, could you take a look at it?

Reply:
Yep, found an bug there